src/routes/github/subscription/github-subscription-router.ts

```diff
@@ -4,7 +4,7 @@ import { GithubSubscriptionDelete } from "./github-subscription-delete";
 import { param } from "express-validator";
 import { returnOnValidationError } from "../../api/api-utils";
 
-export const GithubSubscriptionRouter = Router();
+export const GithubSubscriptionRouter = Router({ mergeParams: true });
```

Comment:
Nested routers need this property to be set otherwise the uuid in the params is undefined.